Description: Use build flags from environment (dpkg-buildflags).
 Necessary for hardening flags.
Author: Simon Ruderich <simon@ruderich.org>
Last-Update: 2012-09-13

--- acl-2.2.51.orig/libmisc/Makefile
+++ acl-2.2.51/libmisc/Makefile
@@ -20,7 +20,7 @@ TOPDIR = ..
 include $(TOPDIR)/include/builddefs
 
 LTLIBRARY = libmisc.la
-LTLDFLAGS =
+LTLDFLAGS = $(LDFLAGS)
 
 CFILES = quote.c unquote.c high_water_alloc.c next_line.c walk_tree.c
 
